reconcile/utils/terrascript_client.py

```diff
@@ -3076,16 +3067,13 @@ def populate_tf_resource_elasticsearch(self, resource, namespace_info):
                 ", and - (hyphen). " +
                 f"{values['identifier']}")
 
-        elasticsearch_version = values.get('elasticsearch_version', 7.7)
-        if not self.validate_elasticsearch_version(elasticsearch_version):
-            raise ElasticSearchResourceVersionInvalidError(
-                f"[{account}] Invalid ElasticSearch version" +
-                f" {values['elasticsearch_version']} provided" +
-                f" for resource {values['identifier']}.")
-
         es_values = {}
         es_values["domain_name"] = identifier
-        es_values["elasticsearch_version"] = elasticsearch_version
+
+        elasticsearch_version = values.get('elasticsearch_version', None)
+        if elasticsearch_version is not None:
+            es_values["elasticsearch_version"] = elasticsearch_version
+
```

Comment:
you can just do:
```
es_values["elasticsearch_version"] = values.get('elasticsearch_version')
```
if its None, terraform will ignore it.

Reply:
TIL